src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -1679,6 +1679,31 @@ def get_index_names(self, axis: int = 0) -> list[Hashable]:
             else self._modin_frame.data_column_pandas_index_names
         )
 
+    def get_index_values(self) -> AnyArrayLike:
```

Comment:
can you reuse `to_numpy` which is used by df.values?

Reply:
no, because that method does not get index values correctly

Reply:
why?

Reply:
was it because of the numpy data format? then can we just call to_pandas on this ?

Reply:
I've removed this now